docs/security.md

```diff
@@ -3,99 +3,85 @@ layout: markdown
+- Service, application, and access logs for sourcegraph.com are stored centrally by Sourcegraph and monitored.
+- Our Incident Response plan is currently under review. It will be publicly available when finalized.
 
-## Data access
+## Sourcegraph on-premise
 
-When running Sourcegraph on your own infrastructure, all application logs are stored locally, and never shared with Sourcegraph. Sourcegraph employees and contractors never have access to your Sourcegraph instance, or any of its data, unless explicitly shared for troubleshooting purposes.
+Sourcegraph on-premise allows you to have the most control over the deployment and security options:
 
-We maintain the following policies for sourcegraph.com data and any data provided via email or other support channels:
+- Sourcegraph instances deployed on-premise do not send any customer code to other servers. Sourcegraph employees have no access to customer data or code.
 
-- Access to all internal systems is protected by multi-factor authentication. Access is restricted to those who require it to perform their job, and is regularly reviewed and revoked upon termination or when no longer needed.
-- Service, application, and access logs for sourcegraph.com are stored centrally by Sourcegraph and monitored.
-- Company policy prevents customer data from being downloaded to portable devices, such as phones, that don't have device management software in place.
-- Sourcegraph deploys Mobile Device management (MDM) for centralized management of remote devices.
-- Laptops have encrypted hard drives.
+- Other than the email address of the initial installer (for customer support, security, and product notification purposes), Sourcegraph never sends any private user data, such as usernames or email addresses, or other specific data to any other servers, including to sourcegraph.com. Learn more in our [pings documentation](https://docs.sourcegraph.com/admin/pings).
 
-## Network security
+- When running Sourcegraph on your own infrastructure, all application logs are stored locally, and never shared with Sourcegraph. Sourcegraph employees and contractors never have access to your Sourcegraph instance, or any of its data, unless explicitly shared for troubleshooting purposes.
 
-When running Sourcegraph on your own infrastructure, you are protected by the network security policies enforced by your infrastructure environment.
+- Authentication via SAML, OAuth, HTTP Proxy auth, and OpenID Connect is configurable. Basic authentication is enabled by default.
 
-On sourcegraph.com, we maintain the following policies:
+- Sourcegraph can be configured to [enforce repository permissions](https://docs.sourcegraph.com/admin/repo/permissions) from connected code hosts. Sourcegraph also exposes a GraphQL API to explicitly set repository permissions.
```

Comment:
@dan-mckean - this is only available to customers on our Enterprise plan.

Reply:
Thanks! Changed it to `Enterprise customers can configure Sourcegraph to enforce...`

Reply:
I'm inclined to not go into detail about plans here. At this stage they're very likely looking at Enterprise anyway - if they just want free, they're likely not checking out security policy in detail. 

Plus it's another place to avoid updating when pricing plans change.

Reply:
But without the distinction it makes it seems like this option is available for all on-prem deployments. If you think the small inaccuracy is ok I'll go back to the previous wording.

Reply:
If you've done it already, leave it. Guessing we'll have to search and change all docs mentions of plans in the event we change things anyway so no big deal :)